lib/seattleflu/api/routes.py

```diff
@@ -3,17 +3,21 @@
 """
 import logging
 from flask import Blueprint, request, send_file
+from flask_deprecate import deprecate_blueprint
 from . import datastore
 from .utils.routes import authenticated_datastore_session_required, content_types_accepted, check_content_length
 
 
 LOG = logging.getLogger(__name__)
 
-api = Blueprint("api", __name__)
+api = Blueprint('new', 'new', url_prefix='/v1')
+old_api = Blueprint('old', 'old', url_prefix='/')
```

Comment:
If we're going to keep this commit (see note above), I think the names `api` and `old_api` would be more future-proof by putting the URL version prefix into the variable name, e.g. `api_v1` and `api_unversioned`. That way when we add `/v2/…` endpoints, we can use `api_v2` and it'll be clear what's what. This also more readily (to me, at least) indicates that the different blueprints are used solely for versioning instead of other common uses like duplicating a set of shared endpoints.

Reply:
I agree with this naming convention.